### PR TITLE
Add combining callback conditions [skip ci]

### PR DESCRIPTION
**Pull request by soartec-lab**
Since there was no explanation of combining callback conditions in Conditional callbacks, it was added.

### CODE REVIEW

File: guides/source/active_record_callbacks.md [hunk 1]
```diff
@@ -338,6 +338,20 @@ class Comment < ApplicationRecord
 end
 ```
 
+### Combining Callback Conditions
+
+when multiple conditions define whether or not a callback should happen, an Array can be used. Moreover, you can apply both :if and :unless to the same callback.
```
Comment (kamipo) [inline guides/source/active_record_callbacks.md]:
```suggestion
When multiple conditions define whether or not a callback should happen, an `Array` can be used. Moreover, you can apply both `:if` and `:unless` to the same callback.
```

File: guides/source/active_record_callbacks.md [hunk 2]
```diff
@@ -338,6 +338,20 @@ class Comment < ApplicationRecord
...
+
+```ruby
+class Comment < ApplicationRecord
+  after_create :send_email_to_author,
+      if: [Proc.new  { |c| c.user.allow_send_email? }, :author_wants_emails?]
+      unless: Proc.new { |c| c.article.ignore_comments? }
+end
+```
+
+The callback only runs when all the :if conditions and none of the :unless conditions are evaluated to true.
```
Comment (kamipo) [inline guides/source/active_record_callbacks.md]:
```suggestion
The callback only runs when all the `:if` conditions and none of the `:unless` conditions are evaluated to `true`.
```

File: guides/source/active_record_callbacks.md [hunk 3]
```diff
@@ -338,6 +338,20 @@ class Comment < ApplicationRecord
...
 ```
 
+### Combining Callback Conditions
+
+when multiple conditions define whether or not a callback should happen, an Array can be used. Moreover, you can apply both :if and :unless to the same callback.
+
+```ruby
+class Comment < ApplicationRecord
+  after_create :send_email_to_author,
+      if: [Proc.new  { |c| c.user.allow_send_email? }, :author_wants_emails?]
```
Comment (kamipo) [inline guides/source/active_record_callbacks.md]:
```suggestion
    if: [Proc.new { |c| c.user.allow_send_email? }, :author_wants_emails?],
```

File: guides/source/active_record_callbacks.md [hunk 4]
```diff
@@ -338,6 +338,20 @@ class Comment < ApplicationRecord
...
+      unless: Proc.new { |c| c.article.ignore_comments? }
```
Comment (kamipo) [inline guides/source/active_record_callbacks.md]:
```suggestion
    unless: Proc.new { |c| c.article.ignore_comments? }
```

**Comment by soartec-lab** [comment]
@kamipo

Thank you for review.
I have committed the proposal.

**Comment by kamipo** [comment]
Thank you for your contribution!
In the future, you can add `[ci skip]` to your *commit message* for documentation-only changes to skip the Travis build.

**Comment by soartec-lab** [comment]
merge Thank you.
I gained good insight through this contribution.
Next time, when updating only doc, [ci skip] will be careful.

**Pull request merged**